pkgs/tools/admin/virtscreen/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, python3Packages, fetchFromGitHub, x11vnc , libGL }:
```

Comment:
Because of https://github.com/NixOS/nixpkgs/pull/53472#discussion_r245493779 you'll need to remove `python3Packages` and add `buildPythonApplication pyqt5 netifaces quamash` here.